app/models/medication.rb

```diff
@@ -1,6 +1,13 @@
 class Medication < ApplicationRecord
-  validates :brand_name, :generic_name, :product_ndc, presence: true
+  validates :brand_name, :product_ndc, presence: true
 
-  has_many :user_medications
-  has_many :users, through: :user_medications
+  has_many :medication_symptoms
+  has_many :symptoms, through: :medication_symptoms
+
+  def save_symptoms(symptoms)
+    symptoms.each do |symptom|
+      new_sym = Symptom.create(description: symptom)
```

Comment:
good call with making this a model method

another way this method could be executed that could address the question I posed to you and @michaeljevans earlier about redundant symptoms: rather than creating a `MedicationSymptom` on line 10, we could use `find_or_create_by(description: <symptom>) and then create a table that stores potential symptoms a user may experience, with each record of a potential symptom not being associated with a medication. This would then make the dropdown menu for the `log` form more straightforward and would eliminate duplicative records of symptoms. We're also not accessing symptoms based on the medication they're associated with, but we are making that association in the database, so this alternative approach would address that as well.

not sure how much fuss this switch would cause but it's worth at least discussing